tests/integ/modin/series/test_str_accessor.py

```diff
@@ -380,6 +384,23 @@ def test_str_split(pat, n):
     )
 
 
+@pytest.mark.parametrize("pat", [None, "a", "|", "%"])
+@pytest.mark.parametrize("n", [None, np.nan, 3, 2, 1, 0, -1, -2])
+@sql_count_checker(query_count=2)
+def test_str_split_expand_true(pat, n):
+    native_ser = native_pd.Series(TEST_DATA)
+    snow_ser = pd.Series(native_ser)
+    native_df = native_ser.str.split(pat=pat, n=n, expand=True, regex=None)
+    snow_df = snow_ser.str.split(pat=pat, n=n, expand=True, regex=None)
+    # Currently Snowpark pandas uses an Index object with string values for columns,
+    # while native pandas uses a RangeIndex.
+    # So we make sure that all corresponding values in the two columns objects are identical
+    # (after casting from string to int).
+    assert all(snow_df.columns.astype(int).values == native_df.columns.values)
+    snow_df.columns = native_df.columns
+    assert_frame_equal(snow_df, native_df, check_dtype=False)
```

Comment:
If you make the column names integers as I suggest in my query compiler comment, you can write

```suggestion
    eval_snowpark_pandas_result(
        *create_test_series(TEST_DATA),
        lambda series: series.split(pat=pat, n=n, expand=True, regex=None)
    )
```

provided that you import `tests.integ.modin.utils.create_test_series` at the top of the file.

Reply:
Good point. Was able to use `eval_snowpark_pandas_result` per your suggestion.